Source/Core/Core/HW/DSPHLE/UCodes/UCodes.cpp

```diff
@@ -51,27 +52,18 @@ UCodeInterface* UCodeFactory(u32 crc, DSPHLE* dsphle, bool wii)
 		INFO_LOG(DSPHLE, "CRC %08x: AX ucode chosen", crc);
 		return new AXUCode(dsphle, crc);
 
-	case 0x6ba3b3ea: // IPL - PAL
-	case 0x24b22038: // IPL - NTSC/NTSC-JAP
-	case 0x42f64ac4: // Luigi's Mansion
-	case 0x4be6a5cb: // AC, Pikmin
-		INFO_LOG(DSPHLE, "CRC %08x: JAC (early Zelda) ucode chosen", crc);
-		return new ZeldaUCode(dsphle, crc);
-
-	case 0x6CA33A6D: // DK Jungle Beat
 	case 0x86840740: // Zelda WW - US
-	case 0x56d36052: // Mario Sunshine
-	case 0x2fcdf1ec: // Mario Kart, Zelda 4 Swords
-	case 0x267fd05a: // Pikmin PAL
-		INFO_LOG(DSPHLE, "CRC %08x: Zelda ucode chosen", crc);
-		return new ZeldaUCode(dsphle, crc);
-
-	// Wii CRCs
-	case 0xb7eb9a9c: // Wii Pikmin - PAL
-	case 0xeaeb38cc: // Wii Pikmin 2 - PAL
-	case 0x6c3f6f94: // Zelda TP - PAL
-	case 0xd643001f: // Mario Galaxy - PAL / Wii DK Jungle Beat - PAL
-		INFO_LOG(DSPHLE, "CRC %08x: Zelda Wii ucode chosen\n", crc);
+	case 0x6ca33a6d: // Zelda TP GC - US
+	case 0xd643001f: // Super Mario Galaxy - US
+	case 0x6ba3b3ea: // GC IPL - PAL
+	case 0x24b22038: // GC IPL - US
+	case 0x2fcdf1ec: // Zelda FSA - US
+	case 0x4be6a5cb: // Pikmin 1 GC - US
+	case 0x267fd05a: // Pikmin 1 GC - PAL
+	case 0x42f64ac4: // Luigi's Mansion - US
+	case 0x56d36052: // Super Mario Sunshine - US
+	case 0x6c3f6f94: // Zelda TP Wii - US
+	case 0xeaeb38cc: // Pikmin 1/2 New Play Control Wii - US
```

Comment:
It looks like the 0xb7eb9a9c CRC disappeared at some point; is that intentional?